### PR TITLE
TDRD-6 Results of metadata upload page

### PR DESCRIPTION
**Pull request by thanhz**

### CODE REVIEW

File: app/controllers/DraftMetadataChecksResultsController.scala [hunk 1]
```diff
@@ -18,7 +21,8 @@ class DraftMetadataChecksResultsController @Inject() (
     val keycloakConfiguration: KeycloakConfiguration,
     val frontEndInfoConfiguration: ApplicationConfig,
```
Comment (vimleshtna) [inline app/controllers/DraftMetadataChecksResultsController.scala:22]: We are not using `frontEndInfoConfiguration` you can remove it from the controller and html page as well

File: app/controllers/DraftMetadataChecksResultsController.scala [hunk 2]
```diff
@@ -27,12 +31,31 @@ class DraftMetadataChecksResultsController @Inject() (
...
+      case FailedValue.value              => failed
+      case CompletedWithIssuesValue.value => DraftMetadataProgress("ERRORS", "red")
+      case CompletedValue.value           => DraftMetadataProgress("IMPORTED", "blue")
+      case _                              => failed
+    } getOrElse failed
+  }
+}
+
+object DraftMetadataChecksResultsController {
+  case class DraftMetadataProgress(value: String, colour: String)
```
Comment (vimleshtna) [inline app/controllers/DraftMetadataChecksResultsController.scala:60]: You can simply create this `DraftMetadataProgress` case class without the `object DraftMetadataChecksResultsController`.

File: test/controllers/DraftMetadataChecksResultsControllerSpec.scala [hunk 3]
```diff
@@ -34,22 +38,6 @@ class DraftMetadataChecksResultsControllerSpec extends FrontEndTestHelper {
   }
 
   "DraftMetadataChecksResultsControllerSpec GET" should {
```
Comment (vimleshtna) [inline test/controllers/DraftMetadataChecksResultsControllerSpec.scala:40]: I know you didn't write this but can you please change "DraftMetadataChecksResultsControllerSpec GET" to `DraftMetadataChecksResultsController GET`?

**Review by vimleshtna — COMMENTED** [review]
Looks good. Just a few minor comments. Thanks
Also, was checking the functionality section of the ticket https://national-archives.atlassian.net/browse/TDRD-6. I think the implementation of the last 3 points is missing in this PR. Can you please check them with Tom if you have to implement them?

**Comment by TomJKing** [comment]
> _Also, was checking the functionality section of the ticket https://national-archives.atlassian.net/browse/TDRD-6. I think the implementation of the last 3 points is missing in this PR. Can you please check them with Tom if you have to implement them?_

@vimleshtna / @thanhz I discussed this with Thanh. The prototype designs have changed since I originally wrote the stories. They are now based on the assumption that what the users uploads will be valid, so there would be no need for the user to download a version of the CSV with the errors in it. Should raise with Emma / Gareth whether they want that functionality implemented now. If so can raise another PR with the necessary changes.

**Pull request merged**